partner_email_check/models/res_partner.py

```diff
@@ -69,6 +71,8 @@ def _normalize_email(self, email):
             result = validate_email(
                 email,
                 check_deliverability=self._should_check_deliverability(),
+                allow_empty_local=test_environment,
```

Comment:
IMO this param is not needed as setting `test_environment` should be enough.

Reply:
@grindtildeath let see in the fixed release, I'll check it later) THX